style.css

```diff
@@ -0,0 +1,1125 @@
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] I am so sorry to raise this but I noticed that your mobile version is tampered with, it's not rendering properly at the moment. Perhaps you need to adjust (index.html line 92 - 94) 👇 

>   <div class="placeholder">
          <img src="images/Img_Placeholder.png" alt="Image Placeholder" />
        </div>